src/page-building/contact/contact-section.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { ContentBlockContainer } from '../../components/layout/content-block-container';
+import { SectionHeader } from '../../new-components/section-header/section-header';
+import { Trans } from 'gatsby-plugin-react-i18next';
+import styled from 'styled-components';
+import { Link } from '../../components/links/links';
+import { theme } from '../../components/layout/theme';
+import { ContactForm } from '../../components/form/contact';
+
+const StyledLink = styled(Link)`
+  color: ${theme.palette.text.link.default};
+
+  &:hover {
+    border-bottom: 1px solid ${theme.palette.text.link.default};
+  }
+`;
+
+export const ContactSection = () => (
+  <>
+    <ContentBlockContainer>
```

Comment:
I'm not sure where to add the `<ContentblockContainer />`. I decided to add it in the subcomponent to make the parent (`contact-page.tsx`) look much cleaner. 
The problem with this is that the `margin-top` is also applied in the storybook, making the story look cluttered. So far there is no consistent solution for this, the other pages do it differently.

Reply:
I have one direction for you: Let's remove them form the individual sections and put them on the outside.
You know it's the correct thing to do when you look at the individual sections in storybook:

See the current storybook of the career page: Every single sections looks pleasant, that's because I did not include the spacing component. With the spacer you will pull in a lot of space at the top. 

I would make the sections "clean" and therefore move that dirty stuff (`ContentBlockContainer`) back into the page please👍